build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildBinPath)\Microsoft.Common.targets" />
+  <PropertyGroup>
+    <BuildDepends>
```

Comment:
@radical: As an *alternative* idea, we could change the `DefaultTargets` in `mono-runtimes.mdproj` to be `_BuildUnlessCached`, and rename `_Build` to `Build`.

This means a "default `xbuild` invocation" would still perform caching, but the "override" behavior would be `xbuild /t:Build`, which might be better?

However, I'm not certain about the ramifications of such a change  along with use of `Microsoft.Common.targets`.